scargo/commands/update.py

```diff
@@ -109,7 +109,6 @@ def pull_docker_image(docker_path: Path) -> bool:
             cmd,
             cwd=docker_path,
             stderr=subprocess.PIPE,
-            check=True,
```

Comment:
I think that removing check is not the best idea, is rather a matter of properly managing the exeption
``` 
logger.warning(e.stderr.decode()) ```

where with the new command there shall be no pulling (or no error) as we create a new project. and for updating the warning instead of the error "Warning: Image cannot be pull as it may not exist or login to registry is required"